appium/webdriver/extensions/screen_record.py

```diff
@@ -29,6 +29,62 @@ class ScreenRecord(webdriver.Remote):
     def start_recording_screen(self: T, **options: Any) -> Union[bytes, str]:
         """Start asynchronous screen recording process.
 
+        +--------------+-----+---------+-----+-------+
+        | Keyword Args | iOS | Android | Win | macOS |
+        +==============+=====+=========+=====+=======+
+        | remotePath   | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | user         | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | password     | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | method       | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | timeLimit    | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | user         | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | forceRestart | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | bugReport    | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | fileFieldName| O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | formFields   | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | bugReport    | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | headers      | O   | O       | O   | O     |
+        +--------------+-----+---------+-----+-------+
+        | videoQuality | O   |         |     |       |
+        +--------------+-----+---------+-----+-------+
+        | videoType    | O   |         |     |       |
+        +--------------+-----+---------+-----+-------+
+        | videoFps     | O   |         |     |       |
+        +--------------+-----+---------+-----+-------+
+        | videoFilter  | O   |         | O   | O     |
```

Comment:
Filters

Reply:
Is `videoFilter` correct? > ref. https://github.com/appium/java-client/blob/ff10d910eae1706d4e0b4888979b9e4ffc91e35d/src/main/java/io/appium/java_client/mac/Mac2StartScreenRecordingOptions.java#L146

Reply:
yeah, videoFilter should be the correct one. sorry for the confusion: https://github.com/appium/appium-mac2-driver/blob/ff143194b199ae08803bce7adc7a51793c810491/lib/commands/record-screen.js#L235